### PR TITLE
Update columns tests

### PR DESCRIPTION
**Pull request by lukewalczak**

### CODE REVIEW

File: test-cases/gutenberg/columns.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 ### Columns block is added with default number of Column
 
 - Add a `Columns` block
+- Press `Cancel` button on layout picker
```
Comment (pinarol) [inline test-cases/gutenberg/columns.md:10]: Is pressing cancel critical for these cases? If selecting one of the layouts don't effect the tests we don't need to state it. It is better if we can keep steps simple, needs less maintenance that way.

Comment (lukewalczak) [reply test-cases/gutenberg/columns.md:10]: It's not critical, I just wanted to be detailed in steps. We can remove it, it shouldn't affect the next steps.

File: test-cases/gutenberg/columns.md [hunk 2]
```diff
@@ -275,3 +292,43 @@ Expected look:
...
+- Expect that fill level in the preview (to the left of a slider) increased
+- Close bottom sheet and select the second column
+- Open its settings
+- Press text input (to the right of a slider)
+- Type value with more than 1 decimal e.g.: `55.55555` and press `Done`
+- Expect to have value with only 1 decimal
+- Close bottom sheet and switch to HTML mode
+- Expect that first column has width equal `90` and the second `55.5`
+- Save the post and open it on web
+- Expect to see columns displayed proportionally to the set width
```
Comment (pinarol) [inline test-cases/gutenberg/columns.md:298]: Too many "Expect"s in one test, could we maybe split this one? I see at least 3 different cases here:

- Check if preview icons are adjusting as we change column percentages
- Check if the column percentage value has proper rounding
- Check if column percentages are getting effective on frontend

Ideally I'd expect that if a test case fails we can link it to a bug report, so better to keep them more atomic.

Comment (pinarol) [reply test-cases/gutenberg/columns.md:298]: A correction for myself: I'd remove `Check if` statements, it is kinda redundant in this context.

Comment (lukewalczak) [reply test-cases/gutenberg/columns.md]: Done, please recheck that

File: test-cases/gutenberg/columns.md [hunk 3]
```diff
@@ -275,3 +275,62 @@ Expected look:
...
+
+### Columns percentage mechanism - column's frontend
+
+- Repeat the steps from [TC021](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/columns.md#tc021) and [TC022](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/columns.md#tc022) within the same `Columns` block
+- Switch to HTML mode
+- Expect that first column has width equal `90` and the second `55.5`
+- Save the post and open it on web
+- Expect to see columns displayed proportionally to the set width
+
+<img src="../resources/columns-layout-picker-flow.gif" width=300 />
```
Comment (pinarol) [inline test-cases/gutenberg/columns.md:319]:
```suggestion
##### TC021

### Columns percentage mechanism - predefined percentage values are rendered correctly on frontend

- Add a `Columns` block
- Expect to see bottom sheet with layout picker
- Choose one of the options e.g: `33/33/33`
- Switch to html mode to see if ratios are correct
- Check the preview to see if the front end reflects the column widths
- Repeat for each layout option

##### TC022

### Columns percentage mechanism - custom percentage values are rendered correctly on frontend

- Add a `Columns` block
- Expect to see bottom sheet with layout picker
- Dismiss the layout picker
- Select the first column, go to its settings
- Use the text input to set its width to a custom value, let's say: `90`
- Set the second column width to another custom value, let's say: `55.5`
- Switch to HTML mode
- Expect that first column has width: `90` and the second one: `55.5`
- Check the preview and expect to see that the front end reflects the column widths
```

Making a new suggestions, it seems to me this way it is more clear what we are testing.

**Pull request merged**